src/layers/src/trip-layer/trip-utils.ts

```diff
@@ -166,3 +167,50 @@ export function getAnimationDomainFromTimestamps(dataToTimeStamp: number[][] = [
     [Infinity, -Infinity]
   );
 }
+
+type GeoJsonFeature = any;
+type CoordsType = number[] & {
+  datumIndex: number;
+  datum: number[];
+};
+
+export function groupColumnsAsGeoJson(
```

Comment:
Nit: Add TSDoc comment to every function, type field doesn't take long but improves the code base quite a bit. At least for new functions this would be really appreciated.